src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentBindLoweringPass.cs

```diff
@@ -596,20 +596,7 @@ private bool TryParseBindAttribute(BindEntry bindEntry, out string valueAttribut
             List<IntermediateToken> valueExpressionTokens,
             List<IntermediateToken> changeExpressionTokens)
         {
-            // Now rewrite the content of the value node to look like:
-            //
-            // BindMethods.GetValue(<code>)
-            valueExpressionTokens.Add(new IntermediateToken()
-            {
-                Content = $"{ComponentsApi.BindMethods.GetValue}(",
-                Kind = TokenKind.CSharp
-            });
             valueExpressionTokens.Add(original);
-            valueExpressionTokens.Add(new IntermediateToken()
```

Comment:
This is the simplification for `bind-Value` - no extra stuff is needed here, we just access the property.